package.json

```diff
@@ -9,6 +9,7 @@
     }
   ],
   "scripts": {
+    "start": "gulp dev",
```

Comment:
The error I noticed (maybe it's not the same as the one you saw) was:

```
os-webview/gulp/tasks/watch.js:32
    gulp.parallel(...watchTasks)
                  ^^^

SyntaxError: Unexpected token ...
    at exports.runInThisContext (vm.js:53:16)
```

But running `nvm use` fixed it (new version of JS that supports `...`)